code/model/DNReferenceList.php

```diff
@@ -59,7 +59,12 @@ public function setLimit($limit) {
 	 * @return array()
 	 */
 	protected function getReferences() {
-		$repository = new Gitonomy\Git\Repository($this->project->LocalCVSPath);
+		try {
+			$repository = new Gitonomy\Git\Repository($this->project->LocalCVSPath);
+		} catch (Exception $e) {
```

Comment:
@stojg If the repo couldn't be cloned or fetched, it should be logged anyway, so it doesn't matter?